### ISSUE TITLE
Galaxy URI should have a dropdown of possible uris

### ISSUE THREAD

**Issue by alexkalderimis** [intermine/im-tables#120]
Rather than just one, there can/should be several.

**Comment by firshta2016** [comment]
I will be working on this issue :) really looking forward to it.

**Comment by firshta2016** [comment]
@yochannah any thoughts and feedback on how this looks? 
![image](https://user-images.githubusercontent.com/24639588/77134440-38a62180-6a24-11ea-91c5-a6db27ad233e.png)

**Comment by yochannah** [comment]
that looks great! If you want a url that isn't in the list, what happends? 

p.s. - keep the two usegalaxy urls but I think we don't want galaxyproject.org ✨

**Comment by firshta2016** [comment]
so the user can type a URL or select from the list of dropdown. Does it need to be reworded differently to communicate that? do you have any suggestions?

**Comment by yochannah** [comment]
nah, that's what I kind of inferred from the text you'd written but since I couldn't test it I asked to be sure :)

**Comment by firshta2016** [comment]
perfect! :) thank you! I will align the text to the left.
